test/parallel/test-tls-set-ciphers.js

```diff
@@ -1,62 +1,93 @@
-// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
Is this OK? I rewrote this test from scratch, but if wanted, I can put the old copyright back in.

Reply:
@nodejs/TSC or @jasnell (who was involved in restoring incorrectly removed copyrights early in the io.js days, IIRC)

Reply:
@nodejs/tsc